allauth/account/tests/test_reset_password.py

```diff
@@ -300,3 +300,28 @@ def _create_user_and_login(self, usable_password=True):
     def _password_set_or_change_redirect(self, urlname, usable_password):
         self._create_user_and_login(usable_password)
         return self.client.get(reverse(urlname))
+
+
+def test_notification_on_password_reset(user_factory, client):
+    user = user_factory(
+        email="john.doe@test.com",
+        username="john.doe",
```

Comment:
Note that you can leave out parameters that are not relevant to your test case. E.g. you can drop username as you are not using it. The factory will populate a value itself.